### DISCUSSION TITLE
Launch "Unrailed!"

### DISCUSSION THREAD

**Discussion by Small-Ku** [RareDevs/Rare#232] [Q&A]
"Unrailed!" is free now in Epic Game Store but it seems that I can't launch it with Rare. It shows the error message below, Do there any ways to launch it?
![image](https://user-images.githubusercontent.com/16464896/183110503-fd134ced-0147-4746-8bd1-58f198d2c051.png)

**Comment by loathingKernel** [comment]
What OS are you on?

**Reply by Small-Ku** [reply]
Windows 11 Pro (21H2 22000.795)

**Reply by loathingKernel** [reply]
@Dummerle Could this be due to the login timing out? Is that fix in the last published version?

**Reply by Dummerle** [reply]
The fix is in the latest release. It works fine on my windows system. 

Maybe you can restart rare and try it again
